ztp/source-crs/optional-extra-manifest/ipsec/README.md

```diff
@@ -0,0 +1,62 @@
+## Configuring IPsec encryption for external traffic
+
+This folder contains the files to configure IPsec encryption for external traffic
+
+* Enable IPsec encryption yaml
+* IPsec endpoint configuration with NMState yml
+* A MachineConfig to import external certs into IPsec NSS and apply the IPsec configuration
+
+Also available in OpenShift docs: [IPsec encryption for external traffic](https://docs.openshift.com/container-platform/4.15/networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.html#nw-ovn-ipsec-external_configuring-ipsec-ovn)
+
+### Prerequisites
+
+* `butane` utility installed
```

Comment:
There should be no pre-req for ZTP. 

ZTP gets applied to new/uninstall server, and there should be no user interruption in the middle of the ZTP until day2 configs are completed.

Reply:
Please note this is an optional feature that requires a pre step, you can automate that as pre step to ZTP or generate the MC once with your certificates and store it for ZTP. 
Please see the official documentation https://docs.openshift.com/container-platform/4.15/networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.html#nw-ovn-ipsec-external_configuring-ipsec-ovn